tests/test_helper/test_npa_constraints.py

```diff
@@ -49,6 +49,7 @@ def cglmp_inequality(dim: int) -> tuple[dict[tuple[int, int], cvxpy.Variable], c
 
 
 # see Table 1. from NPA paper [arXiv:0803.4290].
```

Comment:
This is not a line you added, but seeing it here, I wonder if we want to place this comment inside of the relevant function to which it pertains?

Reply:
Do you have a preference for how the arxiv is added to the docstring? Currently, I simply moved the comment to the docstring without any changes.

Reply:
I think for now we can just leave the arXiv in the docstring in the same format. 

The #191 issue will eventually give us one "reference file to rule them all", but for now, we can save that for another PR.